pyclowder/connectors.py

```diff
@@ -923,31 +927,42 @@ def process_messages(self, channel, rabbitmq_queue):
 
     def status_update(self, status, resource, message):
         super(RabbitMQHandler, self).status_update(status, resource, message)
+
         status_report = dict()
         # TODO: Update this to check resource["type"] once Clowder better supports dataset events
         status_report['file_id'] = resource["id"]
         status_report['job_id'] = self.job_id
         status_report['extractor_id'] = self.extractor_info['name']
         status_report['status'] = "%s: %s" % (status, message)
         status_report['start'] = pyclowder.utils.iso8601time()
+
         with self.lock:
+            # TODO: Remove 'status' from payload later and read from message_type and message in Clowder 2.0
             self.messages.append({"type": "status",
-                                  "status": status_report,
                                   "resource": resource,
-                                  "message": message})
-
-    def message_ok(self, resource):
-        super(RabbitMQHandler, self).message_ok(resource)
+                                  "payload": {
+                                      "file_id":      resource["id"],
```

Comment:
do we need dataset_id?

Reply:
this is both dataset_id and file_id, should fix this in the future, both in clowder and pyclowder.